Framework/Algorithms/src/FitPeaks.cpp

```diff
@@ -312,13 +319,16 @@ void FitPeaks::init() {
       "(highest Y value position) and "
       "the peak width either estimted by observation or calculate.");
 
-  std::string helpgrp("Additional Information");
-
   // additional output for reviewing
   declareProperty(Kernel::make_unique<WorkspaceProperty<API::ITableWorkspace>>(
                       "OutputPeakParametersWorkspace", "", Direction::Output),
                   "Name of workspace containing all fitted peak parameters.  "
                   "X-values are spectra/workspace index.");
+  declareProperty("RawPeakParameters", true,
```

Comment:
Just from a visual standpoint, the last three properties should be (in order) `FittedPeaksWorkspace`, `OutputPeakParametersWorkspace`, `RawPeakParameters`